test/java/org/opendatakit/briefcase/matchers/SwingMatchers.java

```diff
@@ -27,25 +25,6 @@
 
 public class SwingMatchers {
 
-  public static <T> Matcher<JComboBox<T>> hasSelectedItem(T item) {
```

Comment:
new/revisit

Internal test helper. I say we leave it and think why we're not using it before removing it.